signs/config.go

```diff
@@ -1,111 +1,19 @@
 package signs
 
-import (
-	"bufio"
-	"encoding/csv"
-	"errors"
-	"io"
-	"os"
-)
-
 // Config server configuration
 type Config struct {
 	Address            string
 	ScheduleXMLurl     string
 	ScheduleXMLupdate  string
 	ScheduleJSONurl    string
 	ScheduleJSONupdate string
-	TwitterJSONurl     string
```

Comment:
Assuming the twitter is TBD given the changes to twitter's api within the last year?

Reply:
Exactly. The dependencies were deprecated and it was a bit bloated. Will need to tackle the Twitter API changes but really want to understand the goals before we do that. Right now just laser focus on schedule and then sponsors.